include/deal.II/lac/constraint_matrix.templates.h

```diff
@@ -1442,14 +1452,14 @@ namespace internals
       std::vector<Number> vector_values;
 
       /**
-       * Data array for reorder row/column indices. Use a shared ptr to
-       * global_rows to avoid defining in the .h file
+       * Data array for reorder row/column indices.
+       *
```

Comment:
Remove the now empty line. Same in 1462.

Reply:
fixed.